styles.css

```diff
@@ -114,3 +133,356 @@ p {
+  background: rgba(255, 255, 255, 0.24);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 15px;
+  border: #3c3a39 1px solid;
+  margin: 14px 0 24px 24px;
+}
+
+.stack-position1 {
+  margin: 0 0 24px 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  list-style: none;
+  width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: #3c3a39 1px solid;
+  margin-top: 14px;
+  flex-wrap: nowrap;
+}
+
+/* boostrap */
+.stack-positionz {
+  width: 71px;
+  height: 20px;
+  margin: 0 0 24px 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  list-style: none;
+  border: #8993a4;
+  border: #3c3a39 1px solid;
+  margin-top: 14px;
+}
+
+/* mulitply storie stack positioning */
+
+.program-stack1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  width: 25px;
+  height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  padding: 10px 10px;
+  margin: 12px;
+}
+
+.program-stack2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+.program-stack3 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+.program-stack4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+/* ----------------------end------------- */
+.professional-art {
+  margin-bottom: 100px;
+}
+
+.position-items {
+  display: flex;
+  gap: 12px;
+}
+
+.stack-items li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.position-items ul li {
+  display: flex;
+  gap: 12px;
+}
+
+.programs {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.see-project {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+/* backgroud picture and stack */
+
+/* background effect */
+
+.data-arc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  height: 87px;
+  padding-left: 24px;
+  width: 100%;
+}
+
+.see-project-btn {
+  position: relative;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  background: #ff6b00;
+  color: #fff;
+  border: #fff;
+}
```

Comment:
- [ ] Please consider adding a `hover` functionality on the `SEE PROJECT` button as suggested by the previous reviewer.


![Screenshot (91)](https://user-images.githubusercontent.com/78104544/227451959-33972fc9-c4a7-4da4-8b21-82e6a23dabf3.png)

Reply:
hover done

Reply:
resolved

Reply:
hi [braincee](https://github.com/braincee)